### ISSUE TITLE
Support React 0.14

### ISSUE THREAD

**Issue by vicapow** [uber/react-map-gl#5]
Only React 0.13 is currently supported. This is blocked by the latest version of r-dom being published to npm. see: https://github.com/uber/r-dom/blob/master/package.json#L43

cc: @rtsao, @mlmorg

**Comment by mlmorg** [comment]
r-dom `2.1.0` has been published

**Comment by vicapow** [comment]
That was fast. Thanks, @mlmorg!

**Comment by AlejandroSilva** [comment]
I saw a branch with support of react 0.14, can you merge it to master? I really want to use this package, but my app is already using react 0.14

thanks :)

**Comment by vicapow** [comment]
@AlejandroSilva It's my understanding that 0.14 is backwards compatible with 0.13. It just has several console warnings.

**Comment by AlejandroSilva** [comment]
yes it is, using a v0.14 in a project with v0.13 mostly result in a lot of warnings...
But in my case, I have a already working project with v0.14, and trying to install react-map-gl result in a  "unfulfill peer dependencys":

```
asilva@pc ~/w/mototracker> npm install  react-map-gl
...
npm ERR! peerinvalid The package react@0.14.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-redux@4.0.0 wants react@^0.14.0
npm ERR! peerinvalid Peer react-map-gl@0.3.0 wants react@^0.13.0
```

Maybe the 'peer dependency' problem is't not a problem with react-map-gl directly, and some npm trick can be made to make it work, but it will be nice to have the dependencies updated :)

**Comment by vicapow** [comment]
@AlejandroSilva You're welcome try the experimental 0.14 branch directly. It was recently updated by @gnavvy. Not sure when this lands in master but I'd expect within at least the next few weeks.

**Comment by AlejandroSilva** [comment]
@vicapow thanks a lot for the update, I happy will use it until.

**Comment by yuvadm** [comment]
any updates on the 0.14 branch?

**Comment by vicapow** [comment]
@yuvadm Hope to have this out this week but this shouldn't be blocking anyone so long as they use the 0.14 branch.

**Comment by yuvadm** [comment]
@vicapow awesome, thanks!

**Comment by vicapow** [comment]
This has landed with 0.6. Let me know if there's any issues!
